### ISSUE TITLE
No audio playing local MKV files

### ISSUE THREAD

**Issue by Xdpker** [thekirankumar/carstream-android-auto#120]
As title says, video plays flawlessly however audio is missing. Normal youtube works. Videos are H264 codec, MKV container.

**Comment by JustinSelvakumar** [comment]
yes mkv does not work. but it worked when applying parking brake. strange.

**Comment by Xdpker** [comment]
Audio does not work regardless of handbrake position. Do you know what type of video files are supported?

**Comment by JustinSelvakumar** [comment]
it is working in my SIS. file type .mkv, .mp4,

**Comment by Xdpker** [comment]
Hmmm strange. Yeah no audio for me. MK7 GTI with the MIB2 infotainment system.

**Comment by Xdpker** [comment]
I wonder if the phone matters? Like maybe the phone doesn't natively have the decoders so the files don't work? I know in MX player, hardware decoding alone doesn't work for the audio. I need software decoding for sound. I'm on a Oneplus 3T
